en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md

```diff
@@ -738,7 +744,7 @@ Run the following CTL command to change the default location of the export direc
             Default : `/home/.wso2apictl/exported`
             
             
-## Import SSL Certificate for Secure HTTP Communication with API Manager
+## Import the SSL Certificate for Secure HTTP Communication with API Manager
```

Comment:
Since users may need to support multiple certs, shall we make it pulral(_Import SSL Certificates_)?

Reply:
Fixed via https://github.com/wso2/docs-apim/pull/2559/commits/708589ec269c76201da4254ec4d018824fa1cfd8